sdk/pack/metadata.go

```diff
@@ -11,9 +11,10 @@ import (
 // high-level information about the pack which is useful for operators and is
 // also exposed as template variables during rendering.
 type Metadata struct {
-	App          *MetadataApp  `hcl:"app,block"`
-	Pack         *MetadataPack `hcl:"pack,block"`
-	Dependencies []*Dependency `hcl:"dependency,block"`
+	App          *MetadataApp         `hcl:"app,block"`
+	Pack         *MetadataPack        `hcl:"pack,block"`
+	Integration  *MetadataIntegration `hcl:"integration,block,optional"`
```

Comment:
I don't think a `block` can be `optional`. I think if you want to have that block optional, just mark all the fields in `MetadataIntegration` as optional instead.